pkg/networkmanager/firewall.jsx

```diff
@@ -90,14 +97,22 @@ function ServiceRow(props) {
         deleteButton
     ];
 
-    var tabs = [];
+    var simpleBody = [];
     if (props.service.description)
-        tabs.push({ name: _("Details"), renderer: () => <p>{props.service.description}</p> });
+        simpleBody.push(<p key={props.service.id}>{props.service.description}</p>);
+    if (props.service.includes.length > 0) {
+        simpleBody.push(<h5 key="included-services">Included Services</h5>);
+        simpleBody.push(<ul key="services-listing">{props.service.includes.map(s => {
+            let service = firewall.services[s];
+            if (service && service.description)
+                return <li key={service.id}><strong>{service.name}</strong>: {service.description}</li>;
+        })} </ul>);
+    }
 
     return <ListingRow key={props.service.id}
                        rowId={props.service.id}
                        columns={columns}
-                       tabRenderers={tabs} />;
+                       simpleBody={simpleBody.length > 0 ? simpleBody : null} />;
```

Comment:
This means simpleBody is an array of JSX objects. With that they need `key=` properties, otherwise you get React warnings. They may be hidden by the fact that we only enable them and make them fatal on fedora-30 test runs, and this only runs on rhel-8-1 right now.

But instead of key= it may be better to restructure this and avoid the array altogether, split the simpleBody contents in two halves (set conditionally), and render them here in a `<React.Fragment>`.